### PR TITLE
CXX-611 Account for 3.1.4 storage engine swap

### PR DESCRIPTION
**Pull request by acmorrow**

### CODE REVIEW

**Comment by acmorrow** [comment]
I can't patch queue or commit this yet, because the images need to be updated to mongo-orchestration 0.4 (see the ticket linked in CXX-611).

File: .mci.yml [hunk 1]
```diff
@@ -394,12 +419,12 @@ tasks:
         - func: "fetch mongodb"
```
Comment (amidvidy) [inline .mci.yml]: add "use WiredTiger storage" to be explicit.

File: .mci.yml [hunk 2]
```diff
@@ -385,6 +400,16 @@ tasks:
...
+      depends_on:
+        - name: "compile"
+      commands:
+        - func: "set version 3.0"
+        - func: "use WiredTiger storage"
+        - func: "fetch artifacts"
+        - func: "fetch mongodb"
+        - func: "run integration tests"
+
     - name: "integration test latest"
```
Comment (amidvidy) [inline .mci.yml]: change this to "integration test latest with WiredTiger"

**Comment by amidvidy** [comment]
LGTM % nits

File: .mci.yml (hunk 1 above)
Comment (acmorrow) [inline .mci.yml]: I sort of left it off intentionally, the idea being that we only explicitly name the storage engine when it differs from the default on that version.

File: .mci.yml (hunk 2 above)
Comment (acmorrow) [inline .mci.yml]: Same idea: I don't want to specify what latest uses by default: I just want whatever the default is.

**Pull request merged**